src/pages/verify/integrations/firebase.mdx

```diff
@@ -0,0 +1,57 @@
+---
+product: verify
+category: Integrations
+sort: 3
+title: Firebase Authentication with Identity Platform
+subtitle: This tutorial demonstrates how to integrate Criipto Verify with Firebase Authentication with Identity Platform.
+---
+
+import Layout from '../../../layouts/mdx';
+export default Layout;
+
+## Firebase Authentication with OpenID Connect
```

Comment:
![image](https://user-images.githubusercontent.com/699049/218444830-da54f65e-552d-4e22-8116-16df46eff4e5.png)
Section is a little bunched up, can we break it into 2 or 3 paragraphs perhaps?